test/cypress/integration/38-onboarding-modal.test.js

```diff
@@ -1,17 +1,53 @@
-describe('Onboarding modal', () => {
-  it.skip('Shows the onboarding modal on the Collective page after we create a Collective', () => {
-    // test here
-  });
-
-  it.skip('Adds admins to the list when you invite them', () => {
-    // test here
-  });
+import { randomEmail, randomSlug } from '../support/faker';
 
-  it.skip('Removes admins from the list when you click x to remove them', () => {
-    // test here
+describe('Onboarding modal', () => {
+  before(() => {
+    (process.env.ONBOARDING_MODAL = true), cy.login({ redirect: '/create/community' });
```

Comment:
This is an unusual syntax, we should use something more standard.

I'm also not sure if setting `ONBOARDING_MODAL` here has any effect, Cypress and frontend are supposed to have two different processes with separate Javascript contexts.

Reply:
Since Now it is default we need not set it, Removed as well